src/Modal/__snapshots__/Modal.test.js.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`renders without crashing, matches the snapshot 1`] = `null`;
```

Comment:
I'd expect this to have a snapshot of a rendered Modal.... looks like it just rendered `null`?